src/ubuntu/18.04/Dockerfile

```diff
@@ -5,17 +5,17 @@ FROM ubuntu:18.04
 # them in a different layer.
 RUN apt-get update \
     && apt-get install -y wget \
```

Comment:
I think it would make sense to merge the first and second apt-get update and apt-get install calls.

Reply:
I've just mimicked what the other Dockerfiles do. I am fine merging it. Any reason for merging just the first two and not the third one too?

Reply:
With your refactoring, there is no need to retrieve `wget` separately before the other packages.  Additionally there is no point in the second `apt-get update` call because the apt sources hasn't changed.

```
RUN apt-get update \
    && apt-get install -y wget \
    && apt-get update \
    && apt-get install -y \
        clang-3.9 \
        liblldb-3.9-dev \
        lldb-3.9 \
        llvm-3.9 \
        make \
```

Suggestion:
RUN apt-get update \
    && apt-get install -y \
        clang-3.9 \
        liblldb-3.9-dev \
        lldb-3.9 \
        llvm-3.9 \
        make \
        wget \
```

Reply:
Actually, thinking about it more, it seems that if we want to do that, it would make sense to make it as a sweeping change over all the Dockerfiles we have to keep them in line.

Reply:
Ah, I am sorry, I've missed the real point. Ok, I'll do that.

Reply:
Done